email_automation/email-config.py

```diff
@@ -0,0 +1,171 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+import os
+import logging
+import sys
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+#Input the filename as an argument in command line 
+filename_raw = sys.argv[1]
+filename = filename_raw.split(".")[0]
+pool_name = "nrelopenpath-prod-" + filename
+
+current_path = os.path.dirname(__file__)
+config_path = os.path.relpath('../configs/'+ filename_raw, current_path)
+
+#Set up AWS credentials as environment variables + set variables 
+ACCESS = os.environ.get("AWS_ACCESS_KEY_ID")
+SECRET = os.environ.get("AWS_SECRET_ACCESS_KEY")
+TOKEN = os.environ.get("AWS_SESSION_TOKEN")
```

Comment:
Where will we get these keys from? I can see that the KEY_ID and the ACCESS_KEY are likely static, but will users need to obtain a SESSION_TOKEN every time the script is run?

Reply:
I have set the variables by running the command line access exports (on the AWS start page) in my terminal, so other users can do the same since it is a copy-paste operation. The access keys expire after a certain amount of time, so they would need to be re-exported on most executions of this script.

In order to have the script work automatically with GitHub actions, we'll have to create an OIDC provider for GitHub + add an IAM role as detailed here. https://aws.amazon.com/blogs/security/use-iam-roles-to-connect-github-actions-to-actions-in-aws/